components/network/index.ts

```diff
@@ -102,7 +118,7 @@ export class VisNodes extends Vis.DataSet<VisNode> {
   }
 }
 
-export class VisEdges extends Vis.DataSet<VisEdge> {
+export class VisEdges extends DataSet<VisEdge> {
   public constructor(data?: VisEdge[], options?: VisDataSetOptions) {
     super(data, options);
   }
```

Comment:
What's exactly the point of this and the other methods that just take the arguments, pass them to the inherited method and then return it's return value? Would anything change if they weren't there?